data/scripts/creaturescripts/player/update_player_on_advanced_level.lua

```diff
@@ -0,0 +1,15 @@
+local updatePlayerOnAdvancedLevel = CreatureEvent("UpdatePlayerOnAdvancedLevel")
```

Comment:
seria interessante manter a configs:
local config = {
	heal = true,
	save = true,
	effect = false,
}

levando em consideração que alguns servers tem a opção de escolher o que quer deixar habilitado.
obs: existem alguns servers que não regeneram hp e mana ao "upar" então é melhor manter configurável.

Reply:
este é o comportamento padrão da cip, também adicionei a mana que faltava, os efeitos no level up é algo custom e individual que pode ser facilmente aplicando adicionando:
```lua
player:getPosition():sendMagicEffect(math.random(CONST_ME_FIREWORK_YELLOW, CONST_ME_FIREWORK_BLUE))
player:say("LEVEL UP!", TALKTYPE_MONSTER_SAY)
```
Não sei qual decisão os administradores tomaram quanto a conteúdo custom, mas a pack está poluída com varias coisas desnecessárias na qual a maioria das pessoas que utilizam ou não sabem que existem ou não sabem alterar.

Reply:
> este é o comportamento padrão da cip, também adicionei a mana que faltava, os efeitos no level up é algo custom e individual que pode ser facilmente aplicando adicionando:
> 
> ```lua
> player:getPosition():sendMagicEffect(math.random(CONST_ME_FIREWORK_YELLOW, CONST_ME_FIREWORK_BLUE))
> player:say("LEVEL UP!", TALKTYPE_MONSTER_SAY)
> ```
> 
> Não sei qual decisão os administradores tomaram quanto a conteúdo custom, mas a pack está poluída com varias coisas desnecessárias na qual a maioria das pessoas que utilizam ou não sabem que existem ou não sabem alterar.

No one should be held responsible if server administrators cannot read and interpret the config tags.